Release ticloud cli v1.0.0-beta.2 (#18938)
pingcap/docs: tidb-cloud/ticloud-serverless-sql-user-list.md

---
title: ticloud serverless sql-user list
summary: The reference of `ticloud serverless sql-user list`.
---

# ticloud serverless sql-user list

> List TiDB Cloud Serverless SQL users

Reviewer: ```suggestion
List TiDB Cloud Serverless SQL users:
```

```shell
ticloud serverless sql-user list [flags]
```

## Examples

List TiDB Cloud Serverless SQL users in interactive mode:

```shell
ticloud serverless sql-user list
```

List TiDB Cloud Serverless SQL users in non-interactive mode:

```shell
ticloud serverless sql-user list -c <cluster-id>
```

## Flags

In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill them in.

| Flag                    | Description                                                                                                 | Required | Note                                                 |
|-------------------------|-------------------------------------------------------------------------------------------------------------|----------|------------------------------------------------------|
| -c, --cluster-id string | Specifies the ID of the cluster.                                                                            | Yes      | Only works in non-interactive mode.                  |

> | -o, --output string     | Output format, one of ["human" "json"]. For the complete result, please use json format. (default "human"). | No       | Only works in non-interactive mode.                  |

Reviewer 2: ```suggestion
| -o, --output string     | Specifies the output format, one of ["human" "json"]. To get a complete result, use "json" format. (default "human"). | No       | Only works in non-interactive mode.                  |
```
